### ISSUE TITLE
Error when restarting/reloading while the plugin is enabled

### ISSUE THREAD

**Issue by QbDesu** [programmer2514/BetterDiscord-CollapsibleUI#1]
The elements you are trying to access have probably not been created at that point.

This error was caught when I wrapped the initialize function in a try/catch statement
```log
TypeError: Cannot read property 'insertBefore' of null
    at CollapsibleUI.addToolbarIcon (CollapsibleUI.plugin.js:107)
    at CollapsibleUI.initialize (CollapsibleUI.plugin.js:20)
    at CollapsibleUI.start (CollapsibleUI.plugin.js:69)
    at Object.startPlugin (<anonymous>:4:231237)
    at Object.startAddon (<anonymous>:4:231032)
    at Object.loadAddon (<anonymous>:4:192268)
    at Object.loadAddon (<anonymous>:4:229205)
    at Object.loadAllAddons (<anonymous>:4:194368)
    at Object.initialize (<anonymous>:4:189046)
    at Object.initialize (<anonymous>:4:228415)
    at Object.startup (<anonymous>:4:290639)
 ```
 
 
![image](https://user-images.githubusercontent.com/5641607/126811977-56b0cba0-02db-423f-ab95-052b7bb6a189.png)

**Comment by programmer2514** [comment]
Huh, I got that error once during development, but I have been unable to replicate it since.
I refactored some of the code to retry if it fails the first time.
Give it a try and tell me how it works.

**Comment by QbDesu** [comment]
No it results in a StackOverflow/RangeError
If the element isn't there that won't change when you try over and over again. Javascript execution is blocking. Until your script is done with whatever it wants to do nothing changes, Discord won't continue to load, nor will anything be rendered.

**Comment by programmer2514** [comment]
So apparently I don't really know what I'm doing when it comes to when/how BD plugins are loaded. I've reverted the changes for now and I will do more research before trying anything else. Pardon my ignorance, and hopefully I will be able to resolve this soon.

**Comment by programmer2514** [comment]
Keeping this open until I know for _certain_ that this is fixed

**Comment by programmer2514** [comment]
After further testing, this issue is fixed as of v1.1.1
